content/20.treatments.md

```diff
@@ -86,6 +86,68 @@ A list of current studies and their results, using carefully the information req
+Again, conclusions cannot de generalized outside of the study, but the results support further investigation.
+
+A randomized trial from the Shanghai Public Health Clinical Center of 30 COVID-19 patients studied whether HCQ increased rates of virological clearance by respiratory pharyngeal swab on Day 7 post-treatment compared to standard care [how to cite? http://www.zjujournals.com/med/CN/10.3785/j.issn.1008-9292.2020.03.03].
+This trial was published in Chinese with an abstract also in English.
+Only the English abstract was read and interpreted.
+The trial found HCQ showed comparable outcomes to standard care with regard to virological clearance rate, time to virological clearance, and time to body temperature normalization.
+A known weakness is small sample size.
+This problem suggests the study is underpowered to detect potentially useful differences and precludes interpretation of results.
+Additionally, because only the abstract could be read, other design and analysis issues could be present.
+Conclusions should be interpreted very cautiously.
+However, these preliminary negative results do reiterate the need for further study.
+
+A case study treated 11 consecutive patients with HCQ + azithromycin using the same dosing regime reported by Gautret et al [@doi:10.1016/j.medmal.2020.03.006].
+One patient died, two were transferred to the ICU, and one developed a prolonged QT interval leading to discontinuation of HCQ + azithromycin.
+As in the Gautret et al study, virological clearance was assessed at Day 6 post-treatment by nasopharyngeal swabs.
+Of the ten living patients at the time, eight of the patients remained positive for SARS-CoV-2 RNA.
+Interpretation of conclusions are severely limited by lack of comparison group and small sample size.
+However, these results stand in contrast to claims by Gautret et al and illustrates the need for better and further investigation.
+
+There is also conversation of clinical trials showing HCQ is superior to control treatment in inhibiting the exacerbation of pneumonia.
+These conversations cite a Letter to the Editor that makes this claim and refers to 15 pre-registration identifiers from the Chinese Clinical Trial Registry [@doi:10.5582/bst.2020.01047].
+When these identifiers are followed back to the registry, most trials claim to still be recruiting patients and no trials have uploaded data or publications.
+
+###### Summary
+
+In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19. 
+Multiple clinical studies have already been carried out to assess this possibility.
+All current studies are low-quality and have small sample sizes.
+Thus, interpretation is severely limited and must be done cautiously.
+Additionally, disagreements between studies demonstrate there is controversy on the effectiveness of HCQ, as well as HCQ + azithromycin combination therapy.
```

Comment:
Azithromycin probably merits a brief discussion here. Typically thought of as a macrolide antibiotic - so why would it have any efficacy in viral infections? As far as I can tell: Raoult (the dude who wrote that first HCQ paper) used this study as rationale: https://www.pnas.org/content/113/50/14408?ijkey=27c60509502454a97cea735745c9bb158268fffd&keytype2=tf_ipsecsha 

Shows that z-pak may be effective for ZIKV infection. I'll keep looking... there are a few papers that suggest z-pak may augment virus-induced interferon expression but they aren't super compelling

Reply:
Would you be able to draft some sentences on that? I don't think I have the expertise to critically read/synthesize basic immunology!

Reply:
Yep of course. I remember reading the graph and saw the massive combination effect from z-pak and thinking how does that even work... then going on a frantic pubmed search to figure out why. I still think it's pretty sketchy science but there is at least *some* evidence that z-pak may have antiviral effects

Reply:
Hmm... Well I just absent-mindedly accepted it as fact. The beauty of collaboration!

Reply:
Putting these links here as a mental bookmark to come back and write this:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4262884/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5918160/
https://www.ncbi.nlm.nih.gov/pubmed/28099856
https://www.ncbi.nlm.nih.gov/pubmed/27911847

Reply:
```suggestion
Additionally, disagreements between studies demonstrate there is controversy on the effectiveness of HCQ, as well as HCQ + azithromycin combination therapy.
<!-- 
rishirajgoel to add text on Azithromycin
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4262884/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5918160/
https://www.ncbi.nlm.nih.gov/pubmed/28099856
https://www.ncbi.nlm.nih.gov/pubmed/27911847
-->
```

Great discussion.  We could use HTML comments to note these references in the text here.  That way @hufengling could finish this pull request and then @rishirajgoel could make a new one with the additional edits.  These lines should be skipped over in the rendered manuscript.

Reply:
What do you think about adding a section -- either as part of the summary, or before/after it, like "Summary and Further Avenues" or something -- talking about what future research regarding hydroxychloroquine and its derivatives could/should explore? (Unless there is a section elsewhere on this?)

I found this medicinal chemistry paper on hydroxychloroquine alternatives (hydroxyferroquine derivatives, which are organometallic agents) that showed selectivity against SARS-CoV back in 2006. Links here:
https://www.ncbi.nlm.nih.gov/pubmed/16640347
https://pubs.acs.org/doi/full/10.1021/jm0601856#

A section on further research on these antivirals could summarize:
- What information we still need to know about hydroxychloroquine and its effectiveness / what current data are lacking (which you do here already)
- Alternatives to hydroxychloroquine as antivirals that might be worth exploring (^ paper above)
- Maybe... some notes on how antivirals are developed and where we could go from here? There might be another section on this part, not sure!

Also found this paper on antiviral development, which may or may not be worth citing (but putting it here to remind myself and anyone else interested, to think about):
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4765712/
https://www.elsevier.es/es-revista-medicina-universitaria-304-articulo-history-progress-antiviral-drugs-from-S166557961500037X
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2972676/pdf/main.pdf